components/bt/host/bluedroid/api/include/api/esp_hidd_api.h

```diff
@@ -98,6 +98,12 @@ typedef struct {
     uint8_t subclass;                                /*!< HID device subclass */
     uint8_t *desc_list;                              /*!< HID descriptor list */
     int desc_list_len;                               /*!< size in bytes of HID descriptor list */
+
+    // DID Profile SDP
+    uint16_t vendor_id;                              /*!< HID Vendor ID */
+    uint16_t product_id;                             /*!< HID Product ID */
+    uint16_t version;                                /*!< HID Product Version */
+    uint8_t vendor_id_source;                        /*!< HID Country Code */
```

Comment:
According to Device Identification Profile Specification or Device Information Service, the filed `vendor_id_source `  designates which organization assigned the `vendor_id`.

`vendor_id_source `  defined values:
* 0x0001 = Bluetooth SIG assigned Device ID Vendor ID value from the Assigned Numbers document
* 0x0002 = USB Implementer’s Forum assigned Vendor ID value
* 0x0000, 0x0003 – 0xFFFF = Reserved for future use

I think this field can be added into `esp_hid_device_config_t`, since BLE and BR/EDR HID Device both need it for the device information.

Btw, the following code in /esp-idf/components/esp_hid/src/ble_hidd.c shall be modified from 
```C
...
        uint8_t pnp_val[7] = {
            0x02, //0x1=BT, 0x2=USB
            dev->config.vendor_id & 0xFF, (dev->config.vendor_id >> 8) & 0xFF, //VID
            dev->config.product_id & 0xFF, (dev->config.product_id >> 8) & 0xFF, //PID
            dev->config.version & 0xFF, (dev->config.version >> 8) & 0xFF  //VERSION
        };
...
```
to
```C
...
        uint8_t pnp_val[7] = {
            dev->config.vendor_id_source, //0x1=BT, 0x2=USB
            dev->config.vendor_id & 0xFF, (dev->config.vendor_id >> 8) & 0xFF, //VID
            dev->config.product_id & 0xFF, (dev->config.product_id >> 8) & 0xFF, //PID
            dev->config.version & 0xFF, (dev->config.version >> 8) & 0xFF  //VERSION
        };
...
```